script/grblcon.py

```diff
@@ -166,6 +202,9 @@ def __init__(self):
         self._text_entry = urwid.Edit('>>> ')
         self._screen = raw_display.Screen()
 
+        # ADC poll interval
+        self.adc_poll_time = 0.2
```

Comment:
This is only written once. Would it make sense to name it in ALL_CAPS to indicate that it's supposed to be a constant?